spotbugs/etc/findbugsfilter.xsd

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <schema targetNamespace="https://github.com/spotbugs/filter/3.0.0" elementFormDefault="unqualified"
```

Comment:
@NeetigyaPod YOu need the other side of this that I has put on that ticket.  This itself is broken.  The schema location is a list of property=value pairs where the left side is the namespace (what you have now) and the right is the location of the actual xsd.  Note that the namespace is not actually an endpoint.  So the purpose is so you can link to the source right from your IDE.

Reply:
Oh makes sense. Committing the change and pushing, thank you.